README.md

```diff
@@ -28,3 +28,13 @@ start a local network of Hedera Services nodes using Docker Compose.
 The [IntelliJ quickstart](docs/intellij-quickstart.md) covers how to 
 start a local network of Hedera Services nodes from IntelliJ for
 testing and development.
+
+## Contributing
+
+Contributions are welcome. Please see the [contributing](https://github.com/hashgraph/.github/blob/main/CONTRIBUTING.md) guide to see how you can get involved.
```

Comment:
nit: move the `guide` inside the pair of square brackets to make `[contributing guide]`. Or making it `[Contributing Guidelines]` to match with the header in the md file, and to be consistent with the description in `Code of Conduct` below.

Reply:
Sure, done. The Contributor Covenant Code of Conduct is an official community doc, hence why it is capitalized. Our hand written contributing guide is not worthy of capitalization.